omnibus/files/private-chef-cookbooks/private-chef/libraries/preflight_bootstrap_validator.rb

```diff
@@ -92,14 +92,8 @@ def pivotal_key_exists?
   end
 
   def validate_sane_state
-    if pivotal_key_exists?
-      unless secrets_exists?
+    if pivotal_key_exists? && !secrets_exists?
         fail_with err_BOOT006_pivotal_with_no_secrets
-      end
-    else
-      if secrets_exists?
-        fail_with err_BOOT007_secrets_with_no_pivotal
```

Comment:
Continuing the thought, the question there is.. is it an error?  I think it still can be, but it's a condition we might force to occur in the "ctrl+c to do something external and then run reconfigure again" instruction for HA.